src/parseTable.ts

```diff
@@ -193,16 +201,36 @@ function getDateString(dateRaw: string | undefined): string | undefined {
  *
  * First, `/` and `\` are removed since they can't exist in file path.
  * Second, if the escaped url is a empty string or user doesn't specify an
- * url, use page id as the url.
- * @param {Nast.Page} page
- * @param {string} propId
+ * url, generate a slug from the title (if `autoSlug` is `true` in `Config`)
+ * and use it along with page id as the filename,
+ * otherwise just use the page id as is.
+ * @param {string} pageUri
+ * @param {string} customSlug
+ * @param {string} title
+ * @param {Config} config
+ * @returns {string}
+ */
+function getPageUrl(pageUri: string, customSlug: string, title: string, config: Config): string {
+  var url = getSafeUrl(customSlug)
```

Comment:
It's more recommended to use `let` instead of `var` today because it's [block-scoped](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/let#description).